### DISCUSSION TITLE
Error when running node app on vercel

### DISCUSSION THREAD

**Discussion by salted-caramel** [vercel/community#4457] [Help]
### Summary

When I build node app in Vercel, it returns error: Refused to load the font 'https://uv-m2t4jwsyr-salted-caramel.vercel.app/DMSans-Regular.ttf' because it violates the following Content Security Policy directive: "default-src 'none'". Note that 'font-src' was not explicitly set, so 'default-src' is used as a fallback. 

It works locally. 

Appreciate all help!

### Example

https://uv-m2t4jwsyr-salted-caramel.vercel.app/

### Steps to Reproduce

https://uv-m2t4jwsyr-salted-caramel.vercel.app/

**Comment by srcnalt** [comment]
Hi @salted-caramel,

I assume this happens since the font is not under public folder.

With the latest version of NextJS you can use Google Fonts directly in the tsx files. I would really suggest giving this feature a try.

```ts
import { DM_Sans } from 'next/font/google'

const dmSans = DM_Sans({
  weight: '400',
  subsets: ['latin'],
  display: 'swap',
})

export default function MyComponent() {
  return <p className={dmSans.className}>Hello World!</p>
}
```
